src/fidgets/ui/IFrame.tsx

```
@@ -312,6 +318,110 @@ const IFrame: React.FC<FidgetArgs<IFrameFidgetSettings>> = ({
 
+  // Get context directly from SDK - pass through exactly as we receive it
+  const { context: sdkContext } = useMiniAppSdk();
+  
+  // Get real SDK instance for Quick Auth
+  const sdkContextState = useContext(MiniAppSdkContext);
+  const rawSdkInstance = sdkContextState?.sdk;
+  
+  // Get authenticator manager for signing when standalone
+  let authenticatorManager: ReturnType<typeof useAuthenticatorManager> | null = null;
+  try {
+    authenticatorManager = useAuthenticatorManager();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)